src/js/pensions/config/form.js

```diff
@@ -783,13 +792,180 @@ const formConfig = {
+                      adopted: 'Adopted child',
+                      stepchild: 'Stepchild'
+                    }
+                  }
+                },
+                attendingCollege: {
+                  'ui:options': {
+                    expandUnder: 'childDateOfBirth',
+                    expandUnderCondition: (childDOB) => !!isBetween18And23(childDOB)
+                  },
+                  'ui:title': 'Is your child in school?',
+                  'ui:widget': 'yesNo'
+                },
+                disabled: {
+                  'ui:required': (form, index) => _.get(['dependents', index, 'relationship'], form) === 'child',
+                  'ui:options': {
+                    hideIf: (form, index) => _.get(['dependents', index, 'relationship'], form) !== 'child'
+                  },
+                  'ui:title': 'Is your child seriously disabled?',
+                  'ui:widget': 'yesNo'
+                },
+                previouslyMarried: {
+                  'ui:required': (form, index) => _.get(['dependents', index, 'relationship'], form) === 'child',
+                  'ui:options': {
+                    hideIf: (form, index) => _.get(['dependents', index, 'relationship'], form) !== 'child'
+                  },
+                  'ui:title': 'Has your child ever been married?',
+                  'ui:widget': 'yesNo'
+                },
+                married: {
+                  'ui:required': (form, index) => _.get(['dependents', index, 'relationship'], form) === 'child',
+                  'ui:options': {
+                    expandUnder: 'previouslyMarried'
+                  },
+                  'ui:title': 'Are they currently married?',
+                  'ui:widget': 'yesNo'
+                }
+              }
+            }
           }
         },
+        childrenAddress: {
+          path: 'household/dependents/child/address/:index',
+          title: item => `${item.fullName.first} ${item.fullName.last} net worth`,
+          showPagePerItem: true,
+          arrayPath: 'dependents',
+          itemFilter: (item) => !!(item.relationship === 'child'),
+          schema: {
+            type: 'object',
+            properties: {
+              dependents: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    childNotInHousehold: dependents.items.properties.childNotInHousehold,
+                    childAddress: dependents.items.properties.childAddress,
+                    personWhoLivesWithChild: dependents.items.properties.personWhoLivesWithChild,
+                    monthlyPayment: dependents.items.properties.monthlyPayment
+                  }
+                }
+              }
+            }
+          },
+          uiSchema: {
+            dependents: {
+              items: {
+                'ui:title': createDisclosureTitle('fullName', 'Address'),
+                childNotInHousehold: {
```

Comment:
This is pretty awkward, and the option order is No/Yes instead of Yes/No, so maybe we should change the schema to be `childInHousehold`?